blog/_posts/2017-09-26-banshee-bandits-wrap-up.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Banshee Bandits Wrap Up
+layout: post
+created_at: Tues Sept 26 2017
```

Comment:
Hi, thanks for your PR. Your blog post is scheduled to be published on Sat September 30, 2017. Can you update the date everywhere?